server/util-server.js

```diff
@@ -259,7 +261,235 @@ exports.kafkaProducerAsync = function (brokers, topic, message, options = {}, sa
         });
     });
 };
+/**
+ * Sends a SIP REGISTER request
+ * @param {string} sipServer The SIP server to register with
+ * @param {number} sipPort The port of the SIP server
+ * @param {string} transport The transport protocol to use (e.g., 'udp' or 'tcp')
+ * @param {string} username The username for registration
+ * @param {string} userPassword The userPassword for registration
+ * @param {string} version The version of the SIP health monitor
+ * @returns {Promise<object>} The response from the SIP REGISTER request
+ */
+exports.sipRegisterRequest = function (sipServer, sipPort, transport, username, userPassword = undefined, version) {
+
+    // eslint-disable-next-line no-async-promise-executor
+    return new Promise(async (resolve, reject) => {
+        try {
+            const registerRequest = {
+                method: "REGISTER",
+                uri: `sip:${sipServer}:${sipPort}`,
+                headers: {
+                    to: { uri: `sip:${sipServer}:${sipPort}` },
+                    from: { uri: `sip:${username}` },
+                    "call-id": uuid.v4(),
+                    cseq: { method: "REGISTER",
+                        seq: 1 },
+                    "content-length": 0,
+                    contact: { uri: `sip:${username}` },
+                    "User-Agent": "SIP Health Monitor " + version,
+                    "Expires": 60,
+                },
+                transport: transport,
+            };
+            const registrationResponse = await exports.sipRegister(registerRequest);
+            console.log("registrationResponse", registrationResponse);
```

Comment:
Please use our logging functionality instead of console.log as console.log cannot be filtered